src/EnergyPlus/DXCoils.cc

```diff
@@ -14005,6 +14279,1379 @@ Label50: ;
+			DXCoil( DXCoilNum ).ActualSC = 999.0;
+		} // end of on/off if - else
+		
+		DXCoilOutletTemp( DXCoilNum ) = DXCoil( DXCoilNum ).OutletAirTemp;
+		DXCoilOutletHumRat( DXCoilNum ) = DXCoil( DXCoilNum ).OutletAirHumRat;
+		DXCoilFanOpMode( DXCoilNum ) = FanOpMode;
+		DXCoilPartLoadRatio( DXCoilNum ) = PLRHeating;
+		DXCoilTotalHeating( DXCoilNum ) = DXCoil( DXCoilNum ).TotalHeatingEnergyRate;
+		DXCoilHeatInletAirDBTemp( DXCoilNum ) = InletAirDryBulbTemp;
+		DXCoilHeatInletAirWBTemp( DXCoilNum ) = InletAirWetBulbC;
+		
+		// calc secondary coil if specified 
+		if ( DXCoil( DXCoilNum ).IsSecondaryDXCoilInZone ) {
+			CalcSecondaryDXCoils( DXCoilNum );
+		}
+
+	}
+	
+	void
+    CalcVRFIUEvapCondTemp(
+    	int const CoolCoilNum, // the number of the VRF Cooling DX coil to be simulated
+    	int const HeatCoilNum, // the number of the VRF Heating DX coil to be simulated
+    	int const ZoneIndex,   // index to zone where the VRF Terminal Unit resides
+    	Real64 & EvapTemp, // evaporating temperature
+    	Real64 & CondTemp  // condensing temperature 
+    ) {
+	    // SUBROUTINE INFORMATION:
+        //       AUTHOR         Xiufeng Pang, LBNL
+        //       DATE WRITTEN   Feb 2013
+		//       MODIFIED       Feb 2014, Xiufeng Pang, LBNL, Apply the new algorithms from Daikin to simulate the indoor unit
+        //                      Jul 2015, RP Zhang, LBNL, Modify the bounds of the Te/Tc
+        //       RE-ENGINEERED  na
+        
+        // PURPOSE OF THIS SUBROUTINE:
+        // 		 Calculate the VRF IU Te (cooling mode) and Tc (heating mode), given zonal loads.
+        
+        // METHODOLOGY EMPLOYED:
+        // 		 Daikin New VRF Energy Simulation Model.
+        
+        // REFERENCES:
+        // na
+        
+        // USE STATEMENTS:
+        using namespace DataZoneEnergyDemands;
+        
+        // SUBROUTINE ARGUMENT DEFINITIONS:
+        // na
+        
+        // SUBROUTINE PARAMETER DEFINITIONS:
+        // na
+        
+        // INTERFACE BLOCK SPECIFICATIONS
+        // na
+        
+        // DERIVED TYPE DEFINITIONS
+        // na
+        
+        // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
+        Real64 QZnReqSenCoolingLoad;  
+        Real64 QZnReqSenHeatingLoad;
+        Real64 C1Tevap;
+        Real64 C2Tevap;
+        Real64 C3Tevap;
+        Real64 C1Tcond;
+        Real64 C2Tcond;
+        Real64 C3Tcond;
+        Real64 RatedCapCool; // Nominal cooling capacity
+        Real64 RatedCapHeat; // Nominal heating capacity
+        Real64 TairInlet;
+        Real64 Tout;
+        Real64 TcoilIn;
+        Real64 Th2;
+        Real64 Th2min;
+        Real64 Hin;
+        Real64 Win;
+        Real64 BFC;
+        Real64 BFH;
+        Real64 SH;
+        Real64 SC;
+        Real64 Qfan;
+        Real64 Garate; // Nominal air mass flow rate
+        Real64 hADP;
+        Real64 wADP;
+        Real64 hTinwADP;
+        Real64 SHRini;
+        Real64 DeltaT;
+        Real64 RHsat;
+		
```

Comment:
No one will know what these variables represent nor their units. You really should add comments. I can live with some missing, but all are missing here.